WordPress/Classes/Utility/ZendeskUtils.swift

```diff
@@ -492,40 +504,27 @@ private extension ZendeskUtils {
         }
     }
 
-    static func getUserInformationFrom(savedProfile: [String: Any]) {
-
-        if let savedEmail = savedProfile[Constants.profileEmailKey] as? String {
-            ZendeskUtils.sharedInstance.userEmail = savedEmail
-        }
-
-        if let savedName = savedProfile[Constants.profileNameKey] as? String {
-            ZendeskUtils.sharedInstance.userName = savedName
-        }
-    }
-
-    // MARK: - Save to User Defaults
+    // MARK: - User Defaults
 
-    static func saveAccountProfileToUD() {
-        saveProfileToUDFor(key: Constants.zendeskProfileUDKey)
-
-        // Since we have account information, remove no account information.
-        UserDefaults.standard.removeObject(forKey: Constants.zendeskNoAccountProfileUDKey)
-        UserDefaults.standard.synchronize()
-    }
-
-    static func saveNoAccountProfileToUD() {
-        saveProfileToUDFor(key: Constants.zendeskNoAccountProfileUDKey)
-    }
-
-    static func saveProfileToUDFor(key: String) {
+    static func saveSupportInformationToUD() {
```

Comment:
I probably wouldn't abbreviate UserDefaults in the method name here (or in the `getSupportInformation` method either) – I'm not sure I'd guess what UD meant without looking at the implementation :) Also, perhaps it could be `saveUserProfile` / `getUserProfile` instead? "support information" is quite generic.